exporters/zipkin/src/zipkin_exporter.cc

```diff
@@ -93,6 +93,14 @@ sdk::common::ExportResult ZipkinExporter::Export(
   return sdk::common::ExportResult::kSuccess;
 }
 
+void ZipkinExporter::Export(
+  const nostd::span<std::unique_ptr<sdk::trace::Recordable>> &spans,
+  nostd::function_ref<bool(sdk::common::ExportResult)> result_callback)
+  noexcept
+{
+
```

Comment:
For exporters not supporting async, we can internally have these methods call sync interface along with logging a warning. Something like:

```cpp
void ZipkinExporter::Export(
  const nostd::span<std::unique_ptr<sdk::trace::Recordable>> &spans,
  nostd::function_ref<bool(sdk::common::ExportResult)> result_callback)
  noexcept
{
   log_warning();
   auto status = Export(spans);
   result_callback(status);
}

```